tests/test_blueprints.py

```diff
@@ -446,3 +446,43 @@ def handler(request):
         'Sec-WebSocket-Version': '13'})
     assert response.status == 101
     assert ev.is_set()
+
+def test_bp_group():
+    app = Sanic('test_nested_bp_groups')
+    
+    deep_0 = Blueprint('deepest_1', url_prefix='/last')
+    deep_1 = Blueprint('deepest_2', url_prefix = '/last1')
+
+    @deep_0.route('/')
+    def handler(request):
+        return text('OK')
```

Comment:
Can you give each of these tests unique responses, e.g. `OK1`, `OK2`, etc... 

That way we can be sure the correct route is responding for each.

Reply:
Sure thing, done! Also logicked up the path names a bit. (Should I squash into one commit again?)

Reply:
That'd be great. After that, this looks good to me.

Reply:
Awesome, done again :)